addons/calendar/models/calendar_event.py

```diff
@@ -1179,22 +1176,13 @@ def ics_datetime(idate, allday=False):
 
         return result
 
-    def convert_online_event_desc_to_text(self, description):
-        """
-        We can sync the calendar events with google calendar, iCal and Outlook, and we
-        also pass the event description along with other data. This description needs
-        to be in plaintext to be displayed properly in above platforms. Because online
-        events have fixed format for the description, this method removes some specific
-        html tags, and converts it into readable plaintext (to be used in external
-        calendars). Note that for regular (offline) events, we simply use the standard
-        `html2plaintext` method instead.
-        """
-        desc_str = str(description)
-        tags_to_replace = ["<ul>", "</ul>", "<li>"]
-        for tag in tags_to_replace:
-            desc_str = desc_str.replace(tag, "")
-        desc_str = desc_str.replace("</li>", "<br/>")
-        return html2plaintext(desc_str)
+    def get_ics_description(self):
```

Comment:
I think "_" is missing, unless, you want it callable by the client.
And actually, it is used as well as for ical and google calendar, so I wonder if a better name would not be something like: _get_customer_description. 
Idea of "customer" (maybe, there is a better one) is that the description is for the customer here. If we wanted to generate an ical for the employee, we would insert other information (the customer information). What do you think ?